src/bitmessagecurses/__init__.py

```diff
@@ -31,6 +31,8 @@
 from helper_ackPayload import genAckPayload
 from helper_sql import sqlExecute, sqlQuery
 from inventory import Inventory
+
+import helper_sent
```

Comment:
wrong order, should be between lines 21 and 22

Reply:
ok